tests/plugins/colordialog/colordialog.js

```diff
@@ -0,0 +1,56 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: colordialog,wysiwygarea */
+
+( function() {
+	'use strict';
+
+	bender.editor = true;
+
+	bender.test( {
+		assertColor: function( inputColor, outputColor ) {
+			var editor = this.editor;
+
+			editor.once( 'dialogShow', function( evt ) {
+				var dialog = evt.data;
+				dialog.setValueOf( 'picker', 'selectedColor', inputColor );
+				dialog.getButton( 'ok' ).click();
+
+			} );
+
+			editor.getColorFromDialog( function( color ) {
+				resume( function() {
+					assert.areSame( outputColor, color );
+				} );
+			} );
+			wait();
+		},
+
+		'test colordialog add hash to colors 6 digits': function() {
+			this.assertColor( '123456', '#123456' );
+		},
+
+		'test colordialog add hash to colors 3 digits': function() {
+			this.assertColor( 'FDE', '#FDE' );
+		},
+
+		'test colordialog does not add hash 1 digit': function() {
+			// IE8 don't allow on totally wrong values of attributes
```

Comment:
_IE8 doesn't allowing invalid values_ means the attribute will not be set? I think it might be more clear with something like:
`IE8 doesn't allow to use invalid color values.` or `IE8 doesn't set invalid color values.` ?